quay-enterprise/build-support.md

```diff
@@ -45,15 +45,15 @@ Use the environment variable `SERVER` to tell the worker the hostname at which Q
 Here's what the full command looks like:
 
 ```sh
-docker run --restart on-failure -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock quay.io/coreos/quay-builder:v2.3.4
+docker run -it -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock --entrypoint /bin/sh quay.io/coreos/quay-builder:v2.3.4 -c "/usr/sbin/update-ca-certificates ; quay-builder"
```

Comment:
Two changes should be made to this doc:

1. After sentence "Here's what the full command looks like" `docker run -it -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock --entrypoint /bin/sh quay.io/coreos/quay-builder:v2.3.4
`


2. and for self-signed certs`
docker run -it -e SERVER=wss://myquayenterprise -v /path/to/ssl:/usr/local/share/ca-certificates -v /var/run/docker.sock:/var/run/docker.sock --entrypoint /bin/sh quay.io/coreos/quay-builder:v2.3.4 -c "/usr/sbin/update-ca-certificates ; quay-builder"`